scripts/pin_the_slow_way.py

```diff
@@ -31,19 +31,25 @@
 pinned = {
           'boost': 'boost 1.61.*',
           'bzip2': 'bzip2 1.0.*',
+          'ffmpeg': 'ffmpeg 2.8.*',
           'fontconfig': 'fontconfig 2.11.*',
           'freetype': 'freetype 2.6.*',
           'hdf5': 'hdf5 1.8.17|1.8.17.*',
           'icu': 'icu 56.*',
           'jpeg': 'jpeg 9*',
+          'libblitz': 'libblitz 0.10|0.10.*',
+          'libmatio': 'libmatio 1.5.*',
           'libnetcdf': 'libnetcdf 4.4.*',
-          'libpng': 'libpng >=1.6.21,<1.7',
+          'libpng': 'libpng 1.6.23',
```

Comment:
This is being done in PR ( https://github.com/conda-forge/conda-forge.github.io/pull/175 ).